app/frontend/src/components/stories/Avatar.stories.tsx

```diff
@@ -0,0 +1,40 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
+import { Meta, Story } from "@storybook/react/types-6-0";
+import * as React from "react";
+// @ts-ignore
+import imageFile from "../../stories/assets/funnycat.jpg";
+
+import Avatar, { AvatarProps } from "../Avatar";
+
+export default {
+  title: "Components/Avatar",
+  component: Avatar,
+  argTypes: {
+    backgroundColor: { control: "color" },
+  },
+} as Meta;
+
+const user = {
+  avatarUrl: imageFile,
+} as AvatarProps["user"];
+
+const Template: Story<AvatarProps> = (args) => (
+  <>
+    <div style={{ display: "flex" }}>
+      <Avatar {...args} />A default sized Avatar with default sized text.
+    </div>
+    <div style={{ display: "flex", fontSize: "40px" }}>
```

Comment:
How would this work in practice? Since you don't usually set a font size on a div. Just put the avatar inside a `Typography`? Or maybe `<Typography component="div">`?

Reply:
As cascading font-sizes are not part of the design we can use `rem` instead, as Darren proposed in a comment.